pepper-apis/dsm-core/src/main/resources/liquibase/CMI/DDP-7556_mercury_sequencing_roles.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="DDP-7556_mercury_roles" author="pegah">
+        <insert tableName="instance_role">
+            <column name="name" value="has_mercury_order_sequencing"></column>
+        </insert>
+        <insert tableName="access_role">
+            <column name="name" value="mercury_order_sequencing"></column>
+        </insert>
+    </changeSet>
+    <changeSet id="DDP-7556_mercury_sequencing_order_table_id_changes" author="pegah">
+        <addColumn tableName="mercury_sequencing">
```

Comment:
where is that table created?

Reply:
please rename the table to ddp_mercury_sequencing

Reply:
The table creation is already merged for gp testing, in changeset `7556_mercury_sequencing`